bin/github-review.sh

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage() {
+    cat <<EOF
+This script verifies the deployed contracts on a chain ID for a given PR.
+
+USAGE
+    bash ./bin/github-review.sh <PR> <CHAIN_ID> <RPC_URL> <VERSION>
+
+ARGUMENTS
+    PR          The GitHub PR number
+    CHAIN_ID    The chain ID to verify
+    RPC_URL     The RPC URL to use for the chain ID
+    VERSION     The version of the contracts to verify
+
+EXAMPLES
+    bash ./bin/github-review.sh 123 1 https://rpc.ankr.com/eth 1.3.0
+    bash ./bin/github-review.sh 123 1 https://rpc.ankr.com/eth 1.4.1
+EOF
+}
+
+if [[ -n "$(git status --porcelain)" ]]; then
+    echo "ERROR: Dirty Git index, please commit all changes before continuing" 1>&2
+    exit 1
+fi
+if ! command -v gh &> /dev/null; then
+    echo "ERROR: Please install the 'gh' GitHub CLI" 1>&2
+    exit 1
+fi
+if ! command -v cast &> /dev/null; then
+    echo "ERROR: Please install the 'cast' Foundry (forge)" 1>&2
+    exit 1
+fi
+
+pr=0
+chainid=0
+rpc=0
+version=""
+# TODO: "1.0.0" "1.1.1" "1.2.0"
```

Comment:
These are no longer officially supported, so we shouldn't be adding deployments for older Safe versions. I.e. I don't think this is a "TODO".

Reply:
Will remove 👍🏾